Tools/MonoGame.Tools.Tests/BuilderTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿// MonoGame - Copyright (C) MonoGame Foundation, Inc
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.IO;
+using Microsoft.Xna.Framework.Content.Pipeline;
+using MonoGame.Framework.Content.Pipeline.Builder;
+using NUnit.Framework;
+
+namespace MonoGame.Tests.ContentPipeline
+{
+    [TestFixture]
+    public class BuilderTest
+    {
+        string NormalizePath(string path, string append = null)
+        {
+            if (append != null)
+                path = Path.Combine(path, append);
+            return path.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
+        }
+
+        [Test]
+        public void CommandLineParserTests()
+        {
+            var args = ContentBuilderParams.Parse(null);
+            Assert.AreEqual(ContentBuilderMode.None, args.Mode);
+
+            args = ContentBuilderParams.Parse("");
+            Assert.AreEqual(ContentBuilderMode.None, args.Mode);
+
+            args = ContentBuilderParams.Parse("build");
+            Assert.AreEqual(ContentBuilderMode.Builder, args.Mode);
+            Assert.AreEqual(TargetPlatform.DesktopGL, args.Platform);
+            Assert.AreEqual(false, args.CompressContent);
+            Assert.AreEqual(false, args.SkipClean);
+            Assert.IsTrue(Path.IsPathRooted(args.WorkingDirectory));
+            Assert.AreEqual(Directory.GetCurrentDirectory(), args.WorkingDirectory);
+            Assert.AreEqual("Content", args.SourceDirectory);
+            Assert.AreEqual(NormalizePath(Directory.GetCurrentDirectory(), "Content"), args.RootedSourceDirectory);
+            Assert.AreEqual(NormalizePath("bin/Content"), args.OutputDirectory);
+            Assert.AreEqual(NormalizePath(Directory.GetCurrentDirectory(), "bin\\Content"), args.RootedOutputDirectory);
```

Comment:
Hey @harry-cpp .

Any ideas why this line works on Windows but fails on Mac and Linux:
```
  Failed CommandLineParserTests [153 ms]
  Error Message:
     String lengths are both 78. Strings differ at index 70.
  Expected: "...k/MonoGame/MonoGame/Artifacts/Tests/Tools/Release/bin\\Content"
  But was:  "...k/MonoGame/MonoGame/Artifacts/Tests/Tools/Release/bin/Content"
```
Is the normalize code doing `path.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar)` not actually reversing the slashes?

Reply:
It's not, `\` is not an alt directory separator on Unix, unlike Windows passing `\` in a path will just result in a crash on Unix, the only exception being .csproj files as they have compatibility mode.